SOP/SOP-push.adoc

```diff
@@ -0,0 +1,179 @@
+
+==== CLI
+
+. Check that the UnifiedPushServer CR is deployed in the same namespace as the operator
+. Check the environment variables of the Server
+.. Run `oc describe pod/example-unifiedpushserver-<xyz123>`
+.. For further information see https://github.com/aerogear/aerogear-unifiedpush-server#container-configuration.
+.. Note that it will use the values mapped in the Secret created by the operator with the database pod name.
+. Check the environment variables of the Database
+.. Run `oc describe pod/example-unifiedpushserver-postgresql-<xyz123>`
+.. By default it should use the values defined in the Secret created by the operator with the database pod name.
+. Check the logs of the UPS OAuth Proxy Container
+.. Get the service pod name -> `oc get pods | grep unifiedpushserver`
+.. Run `oc logs <service-podname> -c ups-oauth-proxy`
+.. Save the logs by running `oc logs <service-podname> -c ups-oauth-proxy > <filename>.log`
+.. Check if was possible pull the OAuth image
+. Check the logs of the UPS Container
+.. Run `oc logs <service-podname> -c ups`
+.. Save the logs by running `oc logs <service-podname> -c ups > <filename>.log`
+.. See the `pod/example-unifiedpushserver-<xyz123>` logs.
+.. See if was possible to pull the UnifiedPush image.
+. Check the operator pod is present as it is responsible for managing the service pod as described in https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc[UnifiedPushOperatorDown]
+. Check if the secret was created
+.. Go to the namespace of the operator
+.. Run `oc get secrets`
+. Check if the values in the secret are correct.
+.. Run `oc edit secret`
+
+=== UnifiedPushDatabaseDown
+
+Troubleshoot using the following steps via either the console or the cli:
+
+==== Console
+
+. Check that the Database Pod is deployed in the same namespace as the operator
+. Check the environment variables of the Database
+.. Go to : `Applications -> Pods -> example-unifiedpushserver-postgresql-<xyz123> -> Environment Variables`
+.. By default it should use the values defined in the Secret created by the operator with the database pod name.
+. Check the pod logs
+.. Navigate to `Applications -> Pods -> example-unifiedpushserver-postgresql-<xyz123> -> Logs
+. Check the `pod/example-unifiedpushserver-postgresql-<xyz123>` events
+.. See if was possible to pull the Database image.
+. Check the operator pod is present as it is responsible for managing the service pod as described in https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc[UnifiedPushOperatorDown]
+
+==== CLI`
+
+. Check that the Database Pod is deployed in the same namespace as the operator
+.. Run `oc describe pod/example-unifiedpushserver-postgresq-<xyz123>`
+.. By default it should use the values defined in the Secret created by the operator with the database pod name.
+. Check the pod logs
+.. Get the service pod name -> `oc get pods | grep unifiedpushserver-postgresq`
```

Comment:
```suggestion
.. Get the service pod name -> `oc get pods | grep unifiedpushserver-postgresql`
```

Like in the other examples, this will only have `unifiedpushserver` in the name, if the CR name had `unifiedpushserver` in it